app-emulation/qemu/qemu-2.11.0.recipe

```diff
@@ -0,0 +1,180 @@
+	cmd:qemu_system_ppcemb$commandSuffix = $portVersion
+	cmd:qemu_system_s390x$commandSuffix = $portVersion
+	cmd:qemu_system_sh4$commandSuffix = $portVersion
+	cmd:qemu_system_sh4eb$commandSuffix = $portVersion
+	cmd:qemu_system_sparc$commandSuffix = $portVersion
+	cmd:qemu_system_sparc64$commandSuffix = $portVersion
+	cmd:qemu_system_unicore32$commandSuffix = $portVersion
+	cmd:qemu_system_x86_64$commandSuffix = $portVersion
+	cmd:qemu_system_xtensa$commandSuffix = $portVersion
+	cmd:qemu_system_xtensaeb$commandSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	cmd:iasl$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:libcapstone$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libfdt$secondaryArchSuffix
+	lib:libgcrypt$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgnutls$secondaryArchSuffix
+	lib:libgpg_error$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:liblzo2$secondaryArchSuffix
+	lib:libncursesw$secondaryArchSuffix
+	lib:libnettle$secondaryArchSuffix
+	lib:libpixman_1$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libsdl2_2.0$secondaryArchSuffix
+	lib:libsnappy$secondaryArchSuffix
+	lib:libssh2$secondaryArchSuffix
+	lib:libssp$secondaryArchSuffix
+	lib:libtasn1$secondaryArchSuffix
+	lib:libusb_1.0$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	devel:libbz2$secondaryArchSuffix
+	devel:libcapstone$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	devel:libfdt$secondaryArchSuffix
+	devel:libgcrypt$secondaryArchSuffix
+	devel:libGL$secondaryArchSuffix
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libgnutls$secondaryArchSuffix
+	devel:libgpg_error$secondaryArchSuffix
+	devel:libintl$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:liblzo2$secondaryArchSuffix
+	devel:libncursesw$secondaryArchSuffix
+	devel:libnettle$secondaryArchSuffix
+	devel:libpixman_1$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+	devel:libsdl2_2.0$secondaryArchSuffix
+	devel:libsnappy$secondaryArchSuffix
+	devel:libssh2$secondaryArchSuffix
+	devel:libtasn1$secondaryArchSuffix
+	devel:libusb_1.0$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	cmd:awk
+	cmd:cmp
+	cmd:gcc$secondaryArchSuffix
+	cmd:gcov$secondaryArchSuffix
+	cmd:gprof$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:find
+	cmd:iasl$secondaryArchSuffix
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:pod2man
+	cmd:python
+	"
+
+defineDebugInfoPackage qemu$secondaryArchSuffix \
+	"$binDir"/qemu-system-x86_64 \
+	"$binDir"/qemu-system-i386
+
+BUILD()
+{
+	export CFLAGS="-I`finddir B_SYSTEM_HEADERS_DIRECTORY`/capstone"
+
+	runConfigure --omit-dirs "binDir sbinDir" \
+		--omit-buildspec ./configure \
+		--bindir="$commandBinDir" --sbindir="$commandBinDir" \
+		--iasl=`finddir B_SYSTEM_BIN_DIRECTORY`/iasl \
```

Comment:
The configure script already have [a default value for iasl][0]. As long as `iasl` could be found in `PATH`, it should work.

```diff
-		--iasl=`finddir B_SYSTEM_BIN_DIRECTORY`/iasl \
```

[0]: https://github.com/qemu/qemu/blob/v2.11.0/configure#L251